driver/src/test/java/org/neo4j/driver/internal/RoutingDriverTest.java

```diff
@@ -148,8 +149,8 @@ public void shouldFailIfNoRouting()
         // Then
         catch ( ServiceUnavailableException e )
         {
-            assertThat( e.getMessage(),
-                    containsString( "Failed to call 'dbms.cluster.routing.getServers' procedure on server" ) );
+            assertThat( e.getMessage(), containsString( "Failed to run " +
+                    "'Statement{text='CALL dbms.cluster.routing.getServers', parameters={}}' on server." ) );
```

Comment:
Imho user-facing message with `Statement{` does not look pretty. Could we maybe concatenate a prettier message in `GetServersProcedureClusterCompositionProvider` instead of using default `Statement#toString()`?